packages/oas-to-har/src/lib/types.ts

```diff
@@ -3,6 +3,9 @@ export type AuthForHAR = Record<string, number | string | { pass?: string; user?
 export { DataForHAR } from 'oas/types';
 
 export interface oasToHarOptions {
+  // The URL to use for the proxy; defaults to https://try.readme.io
+  proxyAddress?: string;
```

Comment:
can we just consolidate this and `proxyUrl` and we can make this a breaking change? the fact that these are separated out into two options is a bit confusing